src/pages/components/global-alert.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Alerts
+description: >
+  A Global Alert provides system-wide feedback and updates to users.
+package: '@zendeskgarden/react-notifications'
+components:
+  - notifications/src/elements/global-alert/GlobalAlert.tsx
+subcomponents:
+  - notifications/src/elements/global-alert/GlobalAlertButton.tsx
+  - notifications/src/elements/global-alert/GlobalAlertClose.tsx
+  - notifications/src/elements/global-alert/GlobalAlertContent.tsx
+  - notifications/src/elements/global-alert/GlobalAlertTitle.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>Announce new versions of products</li>
+      <li>Inform users of trial status</li>
+      <li>Let users know they are in an editing mode</li>
+      <li>Communicate system maintenance or disconnection from servers</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          For more contextual feedback about an action or task, use an
+          [Alert](/components/notifications)
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+import Default from '../../examples/notifications/global-alert/GlobalAlertDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/notifications/global-alert/GlobalAlertDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Type
+
+Types indicate what kind of message is in the Global alert and help the user understand
+how to respond. They can be info, warning, error, or success.
+
+import Types from '../../examples/notifications/global-alert/GlobalAlertTypes.tsx';
+import TypesCode from '!!raw-loader!../../examples/notifications/global-alert/GlobalAlertTypes.tsx';
+
+<CodeExample code={TypesCode}>
+  <Types />
+</CodeExample>
+
+## How to use it well
+
+### Clearly actionable
```

Comment:
This section should have an associated example, as it is imperative for developers to see how to properly use anchors.

Reply:
@jzempel agreed - FYI, we added a variant with a link example.